src/main/resources/application.yml

```diff
@@ -0,0 +1,29 @@
+server:
+  port:
+
+logging:
+  level:
+    com.knighttodo.knighttodo: INFO
+
+spring:
+  output:
+    ansi:
+      enabled: NEVER
+  datasource:
+    url: jdbc:postgresql://localhost:5432/postgres?currentSchema=neprozorro
+    username: postgres
+    password: 123
+    driver-class-name: org.postgresql.Driver
+  jpa:
+    hibernate:
+      ddl-auto: create
```

Comment:
it's better to use validate by default, other people will change it to create for creating DB

Reply:
Ok